.chloggen/gcp-0482.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: googlecloudexporter
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Google Cloud and GMP exporters are marked as mutating.
```

Comment:
nit: what we usually do is to have 2 changelog entries, one for googlecloudexporter and one for googlemanagedprometheusexporter